### PR TITLE
Substitute warning icon in events view

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

Substitute the warning icon by an info icon.
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7049

### Evidence

image

### Test

- [ ] Checks that events views render `(i)` icon

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by guidomodarelli — APPROVED** [review]
CR :+1: 
TEST :+1: 

image

**Comment by gdiazlo** [comment]
Icon is not centered in the line. It appears to be below the line.

File: plugins/main/public/kibana-integrations/discover/application/components/hits_counter/hits_counter.tsx [hunk 1]
```diff
@@ -43,45 +50,61 @@ export interface HitsCounterProps {
...
-            </EuiToolTip>)}
+            {tooltip && tooltip.content && (
+              <EuiToolTip
+                position={tooltip.position || 'top'}
+                content={tooltip.content}
+              >
+                <EuiIcon
+                  tabIndex={0}
+                  type={tooltip.iconType || 'iInCircle'}
+                  aria-label={tooltip.ariaLabel || 'iInCircle'}
```
Comment (Desvelao) [inline plugins/main/public/kibana-integrations/discover/application/components/hits_counter/hits_counter.tsx:88]: **nitpick**: restore the default value for the `aria-label`.

File: plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.tsx
Comment (Desvelao) [inline plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.tsx:1]: **nitpick**: change the `aria-label` to something such as `Info`.

File: plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx
Comment (Desvelao) [inline plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx:1]: **nitpick**: change the `aria-label` to something such as `Info`.

**Comment by Desvelao** [comment]
# :green_circle: Test

> Maximum documents reduced to the testing
image
image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Comment by guidomodarelli** [comment]
TEST :green_circle:

image

image

**Review by guidomodarelli — APPROVED** [review]
LGTM :green_circle:

**Pull request merged**